### DISCUSSION TITLE
Make PvP more accessible.

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#7652] [Suggestions]
The current PvP system is stifling participation due to high barriers to entry and limited opportunities for progression. Changes are urgently needed:  

- **Lower or Remove Rating Requirements:** Make all S3 gear available with reasonable requirements to lower barriers for all players.  
- **S2 Gear for Honor:** Allow S2 gear to be purchased with honor as a catch-up option.  
- **S1 Gear for Gold:** Make S1 gear purchasable with gold to help new players enter PvP. Most active players already have S1 or S2, leaving ungeared players struggling to compete. Grinding BGs for days just to earn outdated gear is frustrating, especially for those with limited playtime. This change would make PvP more accessible and enjoyable for newcomers.

### **Why This Matters**  
- **No Teams Close to Weapon/Shoulder Ratings:** Even 2 months into S3, top rewards are unattainable, even for the top teams.  
- **Declining PvP Activity:** Participation has dropped significantly; battlegrounds are rarely full, even on weekends, and arena queues for 2s, 3s, and 5s almost never pop.  
- **Impossible Catch-Up:** New players and alts are stuck two seasons behind, worsening the downward spiral.  

These changes would revive PvP, encourage participation, and create a more balanced ecosystem. Let’s act now to save PvP before it declines further!

**Comment by stephenenzor** [comment]
Personally I like the optional “war mode” approach other servers have taken because it gives you a reason/optional reward for PvP flagging (e.g. a 20-30% experience bonus, etc).

**Reply by ChaoticMe** [reply]
make that a separate suggestion so it can gain its own traction.

**Reply by stephenenzor** [reply]
Thank you for the suggestion, I have gone ahead and done that.

**Comment by ChaoticMe** [comment]
The TBC rep blue pvp set is a good option currently in game, but we are at a point where if people can step into good purps, they should. PVE or PVP. Hopefully this is something we can work together to achieve, I like the gear suggestions.

-Fuzz

**Reply by minthes** [reply]
the rep set is locked behind rep and there's no way a new player/alt can compete vs a S3/T6 geared character in blues. I mean it was hard when blue set was current. Let alone now.

**Comment by Never6** [comment]
eyyomylittlamanwannadolamakgora

**Comment by Franaloudzx** [comment]
Completely disagree with these:

"Remove Rating Requirements: Make all S3 gear available without ratings to lower barriers for all players."

I hope no one gets offended by this: the only barrier i see here is the amount of time you guys spend playing battlegrounds, the fact that these ideas are being upvoted more than others (that only ask for some dedication to the arena bracket) only means players here are mostly PvE dedicated, and that's fine, just don't pretend that by that simple rule u just deserve to get the same PvP gear that someone that is actually dedicated / skilled to his class.

"No Teams Close to Weapon/Shoulder Ratings: Even 2 months into S3, top rewards are unattainable, even for the top teams."

If you look closely to SoloQ 3vs3 (the most played bracket so far atm) we can find more than 1 player with 1.8k+ rating, this considering this bracket it's extra challenging since your partners are randomized leading to unbalanced teams. Ppl with 500+ games played. This means basically that the only thing needed is ppl willing to play arena instead of aiming only for gear. And yes these rating were achieved several times in other arena seasons. So instead of focusing on making a PvE friendly system, just ask those who are mainly playing battlegrounds why they keep doing It, when you should be playing arena instead.

Problem here is that none of your ideas are actually looking on fixing "the lack of PvP Activity".

Players who are leading these ideas (daily arena points per bg, more honor gear available, no rating requirements...), they all are yelling the same thing: "don't make me play arenas", when arenas is the peak of PvP and has always been, where true PvP players actually master their class. The truth is that these players are far from being PvP players, the profile of players that upvote these, are always people that are desperate to get the best possible gear with the less amount of braincells (skill) involved.

I will say this always, arena season 1 was so far the most active period of time for PvP in TBC, arena activity was way more active than any other arena season and battlegrounds had their own niche of players (more casual PvE player friendly). But the moment GMS started bringing changes that would only kill 1 bracket in order to make Battlegrounds become the main goal (when by defauly bgs has always been active, simply because of the behavior of these players), PvP just started going down.

Democracy has never been so damaging in history (not gonna mention examples in real Life bcs is not the main topic). In the first place, why would you give some authority to ppl that their biggest achievement in PvP was getting 1000 honor kills. And honestly i always try to support the server by promoting that you can play a progressive PvP experience, but any time they make a change is to make some ppl quit in order to make 80% of playerbase upvote ideas that are oriented to the worst and unskilled players.

1.- 5 minute CD was needed to counter situations like: 2 same teams fighting each other for arena points, until the moment 3 teams were playing this feature would remain, being helpful to create a healthy environment where 3+ teams would get matched creating a sense of real arena bracket where players have way lower chances to face each other permanently (leading to non-competitive strategies such as swapping gear / talents in order to counter that specific team) Consecuences: Harder to wintrade, healthy arena bracket. 

Ppl going against this idea: lack of patience, they just simply wanna play even if It means facing same team over and over, instead of encouraging other players to play.

Consecuences: more ppl playing and dodging other teams in order to not get farmed and getting their so desperately wished arena gear at any cost, just to go back to battlegrounds because yeah, ppl here just get gear to farm others in BG but they don't like to be farmed in arena, awesome level of hipocrisy.

2.- Daily Arena points per BG: this feature was fine with 10 arena points, but the moment they enabled Call to Arms, It became a Big amount of points reaching 45 arena points per win during CTA days, this would make many ppl to just staying away from arena since they would get "enough" arena points to buy their all mighty PvP gear. Giving a total amount right now of never less than 260 arena points.  

"Hey frana but in wotlk we had this feature, what's the problem with It? 

Consecuences: Without rating requirements  giving arena points by doing BGs will make ppl less attracted to play arenas, so again killing 1 bracket in order to make non-pvp players more happy.

3.- Conclusion about the proposal: 
You all should reconsider (and this message is not going personally to you) where is actually the origin of losing PvP population and why players that i actually consider great just leave, if you are constantly leading to ideas more attractive to players who are bad at PvP in general, then that's what you gonna get, a full bunch population of not good players (mainly PvE players) that's what you gonna get. Because if you are good at PvP you will certainly stop playing after some battlegrounds because, the thing is that a true PvP player will always want to progress and find challenges on the way, and that's when one discovers "arenas" where u peak as a player.


My last Call:
First of all thanks to all of you that read til this point, especially to GMs, i'm a bit exhausted but i really care about the server, that's why i took the time to show my disapprovement of what has been done over the time, because all i see is a one sided profile of players that are getting what they want, voting without any consideration to what PvP ideally is and for the true PvP player, because in the first place, if you like PvP u wouldn't give a damn about getting the best gear possible, if you are good and love PvP, you will get this gear in the process, over the time because you are gonna progress and get good at it since you enjoy it.

Sincerely

Franagan.

**Reply by minthes** [reply]
Thank you, Franagan, for sharing your perspective. Your passion for preserving the integrity of PvP is admirable, but I strongly disagree with your conclusions. Your arguments place too much blame on players’ behavior while overlooking the fundamental role of game design in shaping PvP activity. 


### **1. Blaming Players is Misguided**

Your argument assumes that players who prefer battlegrounds over arenas are somehow lazy, unskilled, or lacking commitment. This approach unfairly places the blame on players rather than addressing the root cause of low PvP participation: game design that fails to provide sufficient incentives.

- **Lack of incentives kills participation:** Compare the current state of PvP to the Rank 14 grind during Vanilla. At that time, the game provided a compelling reason to engage in battlegrounds, resulting in vibrant PvP activity. Players filled Alterac Valley matches because there was a tangible reward worth chasing. Today, the system lacks comparable incentives, leading to a decline in participation.
- **Game design shapes player behavior:** Players naturally gravitate toward activities that feel rewarding. If arenas are struggling, it’s not because players are avoiding them out of laziness or unwillingness to improve—it’s because the system doesn’t adequately motivate or reward participation. Blaming players for responding to the incentives (or lack thereof) set by the game is unproductive.

**###2.Gear Should Not Be a Skill Gate**

The argument that PvP players shouldn’t care about gear ignores the practical reality of competitive play. Gear directly impacts performance, and players at lower skill levels are often at a disadvantage when facing geared opponents. Removing rating requirements doesn’t remove the need for skill—it removes an artificial gatekeeping mechanism.

###  **3.PvP Activity Decline is Multifaceted**

Blaming a decline in PvP activity solely on changes like rating-free gear or battleground incentives oversimplifies the issue. The reality is more complex:

- **Accessibility improves activity:** When PvP feels welcoming, more players participate. Higher participation in battlegrounds doesn’t "kill" arenas—it creates a pipeline of players who may later transition to arenas after gaining confidence and experience.

- **Retention matters:** Catering only to the top 20% of PvP players alienates the majority. A thriving PvP community needs a strong foundation, which includes casual players and those learning the ropes.

**### 4. Incentives Drive Activity**

The decline in PvP activity isn’t because players have "gotten lazy" or "don’t want to improve." It’s because the game no longer provides the same compelling incentives as it did during earlier periods.

- Rank 14 grind vs. now: During the Rank 14 grind, PvP activity was at its peak because players had a clear and rewarding goal. Alterac Valley matches were full, and the battleground community thrived. The current system lacks equivalent incentives, which is why PvP feels dead by comparison.

-  Design for engagement: Proposals like daily arena points for battlegrounds and removing rating requirements aren’t about making PvP "easier." They’re about restoring a sense of purpose and reward to the system, encouraging more players to participate.

On a more personal note, my teammate and I absolutely love arena; it's our favorite thing to do in WoW. On another server, we sometimes even do +100 games in a single session, and we’d love to do the same here on Chromie. However, the reality is that it's simply not enjoyable when there aren’t enough players participating. Any game with matchmaking relies on a healthy player base to function properly—that’s just basic math. Unfortunately, Chromie doesn't currently have enough active players engaging in arenas, so we need to focus on creating more and more incentives to get players involved. More players means better matchmaking, better games, and ultimately, a better PvP experience for everyone.

**Reply by Ottawia** [reply]
2k rating reached several times in other season. So in last season there were maybe 5 teams total at that rating and at the end of S2 there were 2 teams above 2k. Gearing is not revard itself (titles and mounts are, that's why they exist).
Soloq and bg have one thing other brackets dont: you dont need your arena partner on and other team on (also 2 specific ppl not randoms) to get in. Players can have life and on our low pop server you need to be chronicaly online to get games and climb rating and it was like that in last season too since its pop issue.
What about challenge in fighting vs same geared players? There is no challenge in winning against players who are 1 or 2 seasons lower than you. I personaly prefer pvp when players are same geared because only skill and comp matter in such a game. And players didnt leave because battlegrounds they left because population is low and its hard to even get weekly games. Because of bugs and bracket where some specs are overpowered and others are trash.

**Comment by vvranceanu** [comment]
wotlk pvp is pointless in 2025 without a draconic anticheat and with a lot of serious bugs present

**Comment by Apathyxia** [comment]
While everyone makes good points here both for and against the suggestion, the main issue still remains in the lack of advertisement for the server. [We currently have ~350 people online during peak times every day](https://chromiecraft.com/apps/players-status-progress/). Out of this maybe 20 people participate in PvP actively. During Call to Arms weekends we clearly see more people queueing up. I do not really see more people from current playerbase to join PvP activities even if we make it more accessible. I'm sure currently active people would play more alts, but I am not convinced that new people would join.

The problem of low PvP participation would somewhat be solved if we did have more people, but more people won't come without advertisement. Further, it is not especially intriguing concept of joining a progressive server mid progression. This will probably change when we actually reach Northrend. 

As much as arena is considered the peak of the PvP, it can't be that when there's no participation. Furthermore, I think we do receive too many arena points from BGs per week. Currently, we can receive 240 arena points weekly from BGs. (45*4 + 20*3). I'd suggest 25 arena points every day for the first win regardless of CtA or not. This would yield 175 are points weekly. 

1. I do not agree with removing rating requirements from gear as it is. I would be in favour of lowering the requirement, and allowing rating in solo queue to fulfil the rating requirement for the items. Those that mainly do PvE and only for like 10 games a week of arena should not be eligible for their BiS weapons with arena points. They should see some effort to achieve those. If rating requirements are removed, I can see this impact negatively on the arena scene, but perhaps BGs would be more in favour. 

2. S2 for honour would give a nice catch-up mechanic for alts and new players, and simiarly S1 for gold. It was a pain to try to learn a new class to PvP as when I'm in blue PvP gear from reputation vendors, and I'm getting one shot by every single warrior in the game. That is extremely unmotivating because _**people on this server especially doing PvP actively are not welcoming at all**_. They will graveyard camp you until you leave, and then complain how no one wants to queue. Not only that, but a person joining a team in green / blue gear almost certainly guarantees a loss for their team in BGs when the games are 4v4 or so. This results in others in the team to complain for the person to get gear etc., but they can't do that when the only option for the gear is the activity "they should not do". 

Another suggestion I do have is that perhaps there could be some scheduled events for normal 2s and 3s that people can prepare for instead of organising games. Maybe there could be bonus honour for arena games played during a given time to include a further incentive. 

This is just some of my rambling and opinions. 

Best regards,
Apatia

**Reply by minthes** [reply]
Yeah on second thought, rating requirement could stay but be lowered. 
I disagree with allowing gear to be obtainable from Soloq. Soloq is already too nice, it gives more AP than 2s, you don't have to bother looking for teammates. And It's strangling regular 2s and 3s while not being as good of a replacement due to several bugs (class stacking, not requiring healer on each side, abusable queue system where it pairs first 3 vs last 3 people who queue). Soloq shouldn't become the default bracket over regular arena and right now it is.

**Reply by Apathyxia** [reply]
You are right that you do not have to search for teammates in soloq, but that also makes it essentially more difficult to gain rating. I do agree that the problems currently in the system should be fixed, and I think there are efforts made to implement some changes on PTR. Although, the system would still work on a order of joining. This needs to be fixed and the operation to put a person on team should only be made when enough people are in the queue to make the queue even pop. It should not be predetermined who will be in your team. 

For so long everyone complained that there's no arenas -> We got solo queue. Now that we got solo queue, people are complaining that there's no 2s or 3s. Frankly, the state of 2s is that if one of the top teams queue, any other team stops queueing. That is the reason why people don't want to queue. Those that actually want to win games and gain rating will dodge the teams they will lose to almost always. And, those with high rating will try to snipe those of lower rating just to get games in and for easy wins. 

Our playerbase isn't large enough to support a healthy arena scene.

**Reply by ChaoticMe** [reply]
_"S2 for honour would give a nice catch-up mechanic for alts and new players, and simiarly S1 for gold. It was a pain to try to learn a new class to PvP as when I'm in blue PvP gear from reputation vendors, and I'm getting one shot by every single warrior in the game. That is extremely unmotivating because people on this server especially doing PvP actively are not welcoming at all. They will graveyard camp you until you leave, and then complain how no one wants to queue. Not only that, but a person joining a team in green / blue gear almost certainly guarantees a loss for their team in BGs when the games are 4v4 or so. This results in others in the team to complain for the person to get gear etc., but they can't do that when the only option for the gear is the activity "they should not do"."_

I strongly disagree that PVPers here are unwelcoming. They'll steamroll you, yes. GY camping is discouraged, I've called it out in General chat before on both sides. that's not what we're about. I don't see it in my matches. That doesn't mean it doesn't happen, but I believe there's an overreaction when it does. We are reaching a gear level at 70 where you aren't guaranteeing a loss for your team if you join in blues, but that's another reason for S1G so that no one has to join in blues/greens. The 4x4 games are not at 70. That's either Arena or leveling BGs. There's a few elitists who grump about people not wearing gear, but crossfaction kind of means the undergeared are free HKs so I don't see as much hateful sentiment here. Straight up, the PVP community here has been far more welcoming than the PVE community, and I've seen a better return on my time investment.

**Reply by ChaoticMe** [reply]
> You are right that you do not have to search for teammates in soloq, but that also makes it essentially more difficult to gain rating. I do agree that the problems currently in the system should be fixed, and I think there are efforts made to implement some changes on PTR. Although, the system would still work on a order of joining. This needs to be fixed and the operation to put a person on team should only be made when enough people are in the queue to make the queue even pop. It should not be predetermined who will be in your team.
> 
> For so long everyone complained that there's no arenas -> We got solo queue. Now that we got solo queue, people are complaining that there's no 2s or 3s. Frankly, the state of 2s is that if one of the top teams queue, any other team stops queueing. That is the reason why people don't want to queue. Those that actually want to win games and gain rating will dodge the teams they will lose to almost always. And, those with high rating will try to snipe those of lower rating just to get games in and for easy wins.
> 
> **Our playerbase isn't large enough to support a healthy arena scene.**

I feel that this is potentially the core problem, the numbers really simply aren't there for Arena. I wish I had a real answer for that, but as a carebear at heart I get it. BUT, it's a core problem we can prop up, and S1G/S2H is a pair of sturdy props.

**Reply by Apathyxia** [reply]
So many early EU morning games are at best 5v5. Most of them are 3v3 or 4v4. These games are 95% decided on gear, or if other team has a healer and one doesn't. More often than not if you have that one blue geared (or full pve geared) person on your team, your team ends up just standing at the GY waiting for the other team to finish the game if they all are properly geared. 

This is just supporting the suggestion that S1 and S2 gears should be way more accessible than what they are now. 

And yes, I do admit to being part of those people that call people out on their gear, especially if they are just deadweight to the team. Because them making the game feel like we are lacking a person makes it very much not enjoyable for me, or others in the team that are competitive. 

When I was learning PvP on my alt, I was honestly feeling horrible when I lacked the gear and didn't feel like I could contribute as much as I should. It was unmotivating to even queue up, but luckily AV weekend did wonders in regards of gear. And, I did get called out several times for my gear when I did anything else but AV. Almost every single BG someone whispered me about it or just called it out on BG chat. Gear was basically blue reputation set and some random pieces I got while levelling to 70. When you are less geared, you will be tunnelled to. There is no reason not to do so as it's just easy to eliminate a player from a fight in 2-3 hits. 

Thus, I can personally say that the PvP community is not especially welcoming here. It's not Warmane level bad, but it's not good. However, I do think it's among the better communities over the dozen of servers I've played on.

**Reply by Ottawia** [reply]
> When I was learning PvP on my alt, I was honestly feeling horrible when I lacked the gear and didn't feel like I could contribute as much as I should. It was unmotivating to even queue up, but luckily AV weekend did wonders in regards of gear. And, I did get called out several times for my gear when I did anything else but AV. Almost every single BG someone whispered me about it or just called it out on BG chat. Gear was basically blue reputation set and some random pieces I got while levelling to 70. When you are less geared, you will be tunnelled to. There is no reason not to do so as it's just easy to eliminate a player from a fight in 2-3 hits.
> 
Tryong to gear new character is pure pain atm especialy if you want to play new class. AV is not na option anymore since we have AV cta onece per 2 months so its like 3 or 4 during entire season. I understand why AV got removed ofc not a single cta bg for entire week is not something any player like to see.
I havent got any wisps with gear blameing yet on any of my characters (and i'm this kind of player who always have some new characters) but i was seeing it many times just like being in blues is not enough stress when you need to care about everthing since every single opponent can 3shoot you.
This is very sad to see. I personaly found CC as exelent server to learning new classes/specs because of low skill gate, we are normies and it makes arena and bg more newbie friendly.

**Comment by Ottawia** [comment]
> * **Lower or Remove Rating Requirements:** Make all S3 gear available with reasonable requirements to lower barriers for all players.
>
2k requirements are makeing gear unobtainable, bearly any players had that high rating in last season and farming rating is painfull because of low population we cant just queue for arena and get games/climb. 
Gear is not prize itself. Mounts and titles are that's why they exist.
Also MMR didnt get reset and its harmfull for players who were participating in arena before. So its more difficult to get gear for players who have skill (i belive) and had high rating before.
However in my opinion rating can stay but should be lowered from 2k to around 1.7 at least. This is rating number which require some commitement but there were much more teams with good players on that rat before and i'm sure they would get it again if arena revive.
> * **S2 Gear for Honor:** Allow S2 gear to be purchased with honor as a catch-up option.
>
100% agree with these arena win should be more based on skill such as line cc etc not on gear. But this is even more important in soloq which is most alive bracket, haveing undeargeared player is increasig chance to lose and it doesnt matter how good are 2 other players or even this one undeargeared. Battleground chats before start is just typing "focus XYZ" now based on gear and we dont even need to check armory or stuff like that server is small enough to memoraise players who have/dont have gear.
Chromie Craft is server for more casual players and it causes CC to be (in my opinion) good place to learn new class/spec. Trying something new can be really fun bit stuggeling on most od bgs because in 4v4 scenario one undeargeared player matters is frustrating. New players/fresh alts are forced to grind honor (for outdated anyway gear) by farming loses this is not good way to getting new players.
